include/revng/PipelineC/Prototypes.h

```diff
@@ -7,29 +7,52 @@
 #include "PipelineC.h"
 
 /**
+ * \page pipelineC_convention PipelineC API conventions
```

Comment:
We're not really using Doxygen for this kind of stuff. If you want to have a "page" we should move this stuff to `mkdocs`.

Can you add a `// TODO: move to mkdocs` so once that gets in we can move it there?

Reply:
Ok, altough it'd be convenient if there was a way to link the mkdocs stuff from doxygen (so that we'd keep the links in the function descriptions)

Reply:
As you wish, but I think our goal is to use Doxygen strictly for documenting the API. For all the rest I want to use `mkdocs`. We can references stuff from Doxygen to mkdocs, eventually.